content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -289,9 +289,18 @@ section.
 
 ## Privileged mode for containers
 
-In Linux, any container in a Pod can enable privileged mode using the `privileged` (Linux) flag on the [security context](/docs/tasks/configure-pod-container/security-context/) of the container spec. This is useful for containers that want to use operating system administrative capabilities such as manipulating the network stack or accessing hardware devices.
-
-If your cluster has the `WindowsHostProcessContainers` feature enabled, you can create a [Windows HostProcess pod](/docs/tasks/configure-pod-container/create-hostprocess-pod) by setting the `windowsOptions.hostProcess` flag on the security context of the pod spec. All containers in these pods must run as Windows HostProcess containers. HostProcess pods run directly on the host and can also be used to perform administrative tasks as is done with Linux privileged containers.
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
```

Comment:
This suggests that Linux privileged containers are not GA in versions < 1.26

Is that a problem?

Reply:
Could remove the feature-state entirely, in other docs it is usually just under the section header so this was an attempt to maintain that trend. Let me know what you think is clearer.

Reply:
I recommend adding two more headings: one for Linux privileged containers and one for Windows privileged containers. Now you have a place for the feature state shortcode that doesn't (shouldn't) confuse people.

If you do this, please consider moving the note to just below the 2nd-level heading.

Reply:
Incorporated into most recent commit.